tools/semaphore-run

```diff
@@ -7,3 +7,14 @@ make V=1 all
 sudo make enable-root-tests
 make -j8 distcheck
 make -j8 check-memory
+
+# TODO: there are currently a lot of pyflakes errors like
+#   'parent' imported but unused
+#   'from testlib import *' used; unable to detect undefined names
+# Filter these out until these get fixed properly.
+out=$(pyflakes test/ test/verify/check-* 2>&1 | grep -Ev "(unable to detect undefined names|defined from star imports|'parent' imported but unused)") || true
```

Comment:
These commands should be placed in a script in ```tools/``` than ```tools/semaphore-run``` then invokes. So that people can do this check locally in the same way that it is done in Semaphore.